product_docs/docs/efm/4/efm_deploy_arch/05_efm_pgbouncer.mdx

```diff
@@ -157,6 +157,7 @@ by root and that user/group/other (0755) has read and execute access. The script
 For the PgBouncer integration, passwordless `ssh` access is required. There are multiple ways 
 to configure `ssh`. Follow your organization's recommended process to 
 configure the passwordless `ssh`. For a quick start, you can also follow this example for configuring passwordless `ssh`.
+The 'efm' user will need to be able to ssh as the user running pgbouncer, i.e. the 'enterprisedb' user.
```

Comment:
```suggestion
The user efm user must be able to ssh as the user running PgBouncer, enterprisedb.
```

Reply:
Conceivably they might install pgbouncer as another user besides 'enterprisedb.' If we say elsewhere that PgBouncer has to be installed/run as enterprisedb then this is ok with me. Definitely need to fix the capitalization for PgBouncer though.

Reply:
How about: The user efm user must be able to ssh as the user running PgBouncer; for example, enterprisedb.

Reply:
Great!